setup.py

```diff
@@ -3,7 +3,7 @@
 setup(
     name="robot",
     version="2023.1",
```

Comment:
Good catch! I would have completely forgotten about `setup.py`, if you want to add that extra bit of polish you could also update the date :)